src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseStudentDetailsEditPageUiTest.java

```diff
@@ -85,9 +85,14 @@ public void testInputValidation() {
             .verifyStatus(String.format(FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE, 
                     "", teamNameFieldName, FieldValidator.REASON_EMPTY, teamNameFieldName, FieldValidator.TEAM_NAME_MAX_LENGTH));
         
-        String invalidStudentName = StringHelper.generateStringOfLength(FieldValidator.PERSON_NAME_MAX_LENGTH + 1);
         String newTeamName = "New teamname";
-        editPage.submitUnsuccessfully(invalidStudentName, newTeamName, null, null)
+        editPage.submitUnsuccessfully("", newTeamName, null, null)
+            .verifyStatus(String.format(FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE,
+                    "", personNameFieldName, FieldValidator.REASON_EMPTY, personNameFieldName, FieldValidator.PERSON_NAME_MAX_LENGTH));
+        
+        String invalidStudentName = StringHelper.generateStringOfLength(FieldValidator.PERSON_NAME_MAX_LENGTH + 1);
+        //Without changing team name, the confirmation will not be sent and if it is not handled correctly, timeout error will occur 
```

Comment:
This explanation should actually go below for the method <code>fillStudentDetailsForm</code>